TradeItIosTicketSDK2/TradeItBrokerLogoService.swift

```diff
@@ -9,51 +10,73 @@ class TradeItBrokerLogoService {
     private var cache: [String: [TradeItBrokerLogoSize: UIImage]] = [:]
 
     func loadLogo(
-        forBroker broker: TradeItBroker,
+        forBrokerId brokerIdOptional: String?,
         withSize size: TradeItBrokerLogoSize,
         onSuccess: @escaping (UIImage) -> Void,
         onFailure: @escaping () -> Void
```

Comment:
Since this is an internal class we could have actually changed this interface to just return a promise. Might be a fun thing to do if you have 20 min spare.